src/logger.rs

```diff
@@ -1,33 +1,31 @@
-use fern;
 pub use log::LevelFilter;
-use std::env;
-use std::io;
+
+use fern;
+use std::{env, io, path::PathBuf, str::FromStr};
 
 /// Logger configuration object.
-#[derive(Clone, Copy)]
+#[derive(Clone)]
 pub struct LoggerConfig {
```

Comment:
Could add a `allow_env_override` or something like that so @Khionu is happy with this, too ;)